src/Sdk/BaseSdk.ts

```diff
@@ -0,0 +1,43 @@
+import { Entity, HeadersMap } from '../types';
+
+import { DEFAULT_USER_AGENT } from './constants';
+
+export interface Options {
+    accessToken: string;
+    baseUrl: string;
+    headers: HeadersMap;
+}
+
+/**
+ * Remove heading and trailing slashes.
+ * Examples:
+ *  - https://example.com/ -> https://example.com
+ *  - /v2/path -> v2/path
+ *  - /v2/path/ -> v2/path
+ */
+const sanitizeUrl = (url: string): string => url.replace(/^\/|\/$/g, '');
```

Comment:
How we could change the code so that:
- somewhat complex regular expression is broken into 2 simpler regular expressions (no `|`, no `g`)
- code is expressive enough so it does not need comments
- we're left with less lines than initially

```suggestion
const removeLeadingSlash = (url: string) => url.replace(/^\//, '');
const removeTrailingSlash = (url: string) => url.replace(/\/$/, '');
const sanitizeUrl = (url: string) => removeTrailingSlash(removeLeadingSlash(url));
```

Reply:
Or we can keep the all-in-one regex, but rename the function to `trimSlashes()` -- so it's obvious what it does.